addons/stock_dropshipping/report/stock_report_views.xml

```diff
@@ -0,0 +1,13 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<odoo>
+    <record id="action_report_dropship" model="ir.actions.report">
+        <field name="name">Delivery Slip</field>
+        <field name="model">stock.picking</field>
+        <field name="report_type">qweb-pdf</field>
+        <field name="report_name">stock.report_deliveryslip</field>
+        <field name="report_file">stock.report_deliveryslip</field>
+        <field name="print_report_name">'Delivery Slip - %s - %s' % (object.purchase_id.dest_address_id.name or '', object.name)</field>
+        <field name="binding_model_id" ref="model_stock_picking"/>
+        <field name="binding_type">report</field>
+    </record>
```

Comment:
![image](https://github.com/odoo/odoo/assets/18055894/03a9206d-9189-4168-b081-d67699a8d1d1)
I don't think that's something we want. Why do you have to duplicate it, especially if that's to point to the stock report anyway? :thinking:

Reply:
I created that action to change the file name for the dropship, but now I have done it by overwriting the file name in dropship. This way, it won't be duplicated now.